src/karmen_backend/openapi/swagger.yaml

```diff
@@ -0,0 +1,1544 @@
+                  type: string
+                  format: password
+                password_confirmation:
+                  type: string
+                  format: password
+      responses:
+        '204':
+          description: 'User activated'
+        '404':
+          description: Activation failed
+  '/users/me/request-password-reset':
+    post:
+      operationId: server.routes.users_me.request_password_reset
+      summary: 'Request password reset for user'
+      tags:
+        - users-me
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+      responses:
+        '202':
+          description: 'Password reset requested'
+        '400':
+          description: Request not valid
+  '/users/me/reset-password':
+    post:
+      summary: 'Process requested password reset'
+      operationId: server.routes.users_me.reset_password
+      tags:
+        - users-me
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                pwd_reset_key:
+                  type: string
+                password:
+                  type: string
+                  format: password
+                password_confirmation:
+                  type: string
+                  format: password
+      responses:
+        '204':
+          description: 'Password changed'
+        '400':
+          description: 'Password reset unsuccessfull'
+  '/users/me/authenticate':
+    post:
+      summary: 'Login via username and password and get refresh token'
+      operationId: server.routes.users_me.authenticate
+      tags:
+        - users-me
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                password:
+                  type: string
+                  format: password
+
+      responses:
+        '200':
+          description: 'Login successfull'
+        '401':
+          description: Wrong login
+  '/users/me/authenticate-fresh':
+    post:
+      summary: 'Get new access token'
+      operationId: server.routes.users_me.authenticate_fresh
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                email:
+                  type: string
+                  format: email
+                password:
+                  type: string
+                  format: password
+      tags:
+        - users-me
+      responses:
+        '200':
+          description: 'Ok'
+        '401':
+          description: 'Unauthorized'
+
+  '/users/me/authenticate-refresh':
+    post:
+      summary: 'Refresh fresh token'
```

Comment:
Refresh token

Reply:
We have `access tokens` with lifespan of 15 minutes to auth user actions, and `refresh tokens` with longer lifespan of 7 days, to get access token